examples/AN00246_xua_example/README.rst

```diff
@@ -1,27 +1,66 @@
+:orphan:
+
 .. |I2S| replace:: I\ :sup:`2`\ S
 
-Simple USB Audio Device using lib_xua
-=====================================
 
-Summary
--------
+##############################################
+AN00246: Simple USB Audio Device using lib_xua
+##############################################
+
+:vendor: XMOS
+:version: 1.0.0
+:scope: Example
+:description: Simple USB Audio device using lib_xua
+:category: General Purpose
+:keywords: USB Audio, |I2S|
```

Comment:
I think keywords cannot have substitutions since the website doesn't use xmosdoc when grabbing.  Just I2S is fine

Reply:
fixed it